### ISSUE TITLE
Uncaught (in promise) RuntimeError: unreachable executed

### ISSUE THREAD

**Issue by amiyatulu** [Synphonyte/leptos-use#73]
When I run this simple code, it doesn't give any error on compile, gives error on rendering in browser.
Uncaught (in promise) RuntimeError: unreachable executed

```rust
use crate::components::accounts::accounts_store::AccountStore;
use crate::components::navigation::nav::Nav;
use leptos::*;
use leptos_use::storage::{use_local_storage, JsonCodec};



#[component]
pub fn AddAccount() -> impl IntoView {
    let (account_store, set_account_store, reset_acccount_store) =
        use_local_storage::<AccountStore, JsonCodec>("account-store-state");

    view! {
        <>
        <Nav/>
     </>
}
}
```

Error goes away when i remove
```rust
 let (account_store, set_account_store, reset_acccount_store) =
        use_local_storage::<AccountStore, JsonCodec>("account-store-state");
```

```toml
[dependencies]
leptos = { version = "0.6.3", features = ["csr", "nightly"] }
leptos-use = {version="0.9.0",  features = ["serde"] }
leptos_router = { version = "0.6.3",  features = ["csr", "nightly"]}
serde = "1.0.196"
```

**Comment by amiyatulu** [comment]
Seems you are updating the repository, example folder also giving error. 
```rust
error[E0432]: unresolved import `leptos_use::storage::JsonCodec`
 --> use_storage/src/main.rs:3:46
  |
3 | use leptos_use::storage::{use_local_storage, JsonCodec};
  |                                              ^^^^^^^^^ no `JsonCodec` in `storage`
  |
  = help: consider importing this struct instead:
          leptos_use::utils::JsonCodec
```

**Comment by maccesch** [comment]
Thanks for reporting this! It is now fixed in e5383c6.

`JsonCodec` lives now in `leptos_use::utils`.
